### PR TITLE
RemoteID: Add an AEROENTRY company

### PR DESCRIPTION
**Pull request by muramura**
I would add AEROENTRY, a company that develops and sells remote ID devices to Japanese specifications.

URL
https://aeroentry.co.jp/info/product/228/

### CODE REVIEW

File: common/source/docs/common-remoteid.rst [hunk 1]
```diff
@@ -20,6 +20,7 @@ MAVLink enabled devices that connect to the autopilot:
 Stand-alone devices:
 
 - `Aerobits idME+ <https://www.aerobits.pl/product/idme-remoteid/>`__
+- `AEROENTRY remote id (Japan) <https://aeroentry.co.jp/info/product/228/>`__
```
Comment (Hwurzburg) [inline common/source/docs/common-remoteid.rst:23]: link gives me a 404 not found

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-remoteid.rst (hunk 1 above)
Comment (muramura) [reply common/source/docs/common-remoteid.rst:23]: @Hwurzburg san.  I converted it to an HTML file in PANDOC and successfully displayed the page.

**Comment by Hwurzburg** [comment]
I can view it now....which region does this claim compatibility with? Japan, Europe, or USA?

**Comment by muramura** [comment]
@Hwurzburg san.  Japanese specifications.

**Comment by Hwurzburg** [comment]
thanks...merging

**Pull request merged**